proposals/records/1-index-scaling.md

```diff
@@ -0,0 +1,183 @@
+# IndexNode Scaling: Coordinator layer
+
+This proposal focuses on moving the shard replica management, indexing and search load balancing
+to a seperate Coordinator layer.
+
+This proposal differs from https://github.com/nuclia/nucliadb/pull/722 in that it keeps all the
+same core concepts of the database engine in place while still allowing us to scale.
+
+
+*Goals*:
+- Allow us to dynamically scale reads/writes to real load, not shard size or count
```

Comment:
What metric do you think we could use as real load?

Reply:
Added some points. Does that help?